### DISCUSSION TITLE
What will happen with Mythic Botany's new Alfheim dimension? There seemed to be a plan in Projects to create a custom one.

### DISCUSSION THREAD

**Discussion by pm065** [NillerMedDild/Enigmatica6#2155] [Q&A] [answered]
Honestly, I'm not so sure if I even like it myself. I was wondering if it was going to be disabled...?

**Comment by NielsPilgaard** [comment, marked as answer]
We're not sure yet, if Mythic Botany's Alfheim is good enough we'll just use that :)

**Reply by pm065** [reply]
My personal opinion on it is that the way of getting there is pretty creative and neat, but once you reach the dimension you feel a little... underwhelmed. It doesn't have particularly spectacular terrain, nor does it have many biomes, but I guess it is new, so maybe it just needs time.
